README.md

```diff
@@ -1,8 +1,27 @@
 # Pharmacy Information Management System
 
+## Overview of repository
+
+The Pharmacy Information Management System (PIMS) is a mock pharmacy system written in TypeScript/JavaScript/MongoDB that handles FHIR R4 (4.0.1), NCPDP Script, and proprietary data exchanges between the [request-generator](https://github.com/mcode/request-generator/), [test-ehr](https://github.com/mcode/test-ehr/), and [rems-admin](https://github.com/mcode/rems-admin/). This repository consists of a frontend service and a backend service.
```

Comment:
I'm guessing the JavaScript code is in JavaScript and not TypeScript since we're using mongoose (?), but I don't think it's accurate to call this a purely TypeScript repository. It would be nice if we had types in the HTTP routes and mongoose schemas though.

Reply:
okay fair enough